.github/workflows/wasm-publish.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: Wasm-Dependency
+on:
+  push:
+    tags: 'release-*'
+
+jobs:
+  publish:
+    name: Publish Wasm Dependecy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.16.1'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - run: |
+          cd src/js
```

Comment:
You can use the [working-directory directive](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun) instead of `cd ...`. 
That'll make the actually ran commands simpler.